Avoid negative wording when describing Cython. (#1059)
pypa/packaging.python.org: source/guides/packaging-binary-extensions.rst

  compile most Python code to C extension modules. The initial compilation
  provides some speed increases (by bypassing the CPython interpreter layer),
  and Cython's optional static typing features can offer additional

>   opportunities for speed increases. Using Cython still has the disadvantages
>   that binary extensions generally impose on distributing the resulting application,

Reviewer: I think this might be slightly clearer:

```suggestion
  opportunities for speed increases. Using Cython still carries the
  `disadvantages`_ associated with using binary extensions,
```

wrapper modules up to date.

* In addition to being useful for the creation of accelerator modules,
  `Cython <https://cython.org/>`__ is also widely used for creating wrapper
  modules for C or C++ APIs. It involves wrapping the interfaces by
  hand, which gives a wide range of freedom in designing and optimising
  the wrapper code, but may not be a good choice for wrapping very
  large APIs quickly. See the
  `list of third-party tools <https://github.com/cython/cython/wiki/AutoPxd>`_

>   for automatic wrapping with Cython. Several Python implementations are
>   supported that provide a CPython-like C-API, most notably PyPy and Pyston.

Reviewer: Assuming this is accurate, I think this is clearer:

```suggestion
  for automatic wrapping with Cython. It also supports performance-oriented
  Python implementations that provide a CPython-like C-API, such as PyPy
  and Pyston.
```

Also, maybe it's worth linking to PyPy and Pyston?

Author: > Also, maybe it's worth linking to PyPy and Pyston?

Well, there is a whole paragraph on PyPy right in the section above. People who don't bother looking up the acceleration section will probably already know the options, or will not care.
